Gopkg.toml

```diff
@@ -1,25 +0,0 @@
-# Gopkg.toml example
```

Comment:
It is great that you have get rid of the dependencies.

It would be nice if you make this repo a go module, so people can start depending on it using `go mod`.